documents/affiliated_package_review_process.md

```diff
@@ -151,14 +152,15 @@ Reviewing a package involves reading over the documentation, having a look at
 the code to assess for example readability (but not read it line by line),
 checking how well the package is tested, as well as assessing its integration
 with the Astropy and wider astronomy ecosystem. Most packages can be reviewed
-within an hour. If you accept this review, we will send you a detailed list of
-criteria to check for the package as well as how to submit your review.
+within an hour and reviews are anonymous by default. If you accept this review,
+we will send you a detailed list of criteria to check for the package as well as
+how to submit your review.
 
 Could you reply to this email to let us know if you would be willing to review
 this package?
 
 Thanks!
-<coordinator name>
+*coordinator name*
```

Comment:
```suggestion
{coordinator_name}
```
and how about the same for "potential reviewer" above?  Then it's trivial to turn this into a `.format` or jinja template... And I fine `*` difficult to remember for replacing because it look like it's supposed to be emphasis